opencsp/app/sofast/test/test_SystemSofastFringe.py

```diff
@@ -14,6 +15,8 @@
 
 @pytest.mark.no_xvfb
 def test_SystemSofastFringe():
+    plt.close('all')

```

Comment:
Interesting. Why is this needed here?

If this is a fix to plots not being closed in other unit tests, then that is a problem with the other unit tests and should not be spilling over into this integration test.  That being said, for the sake of not blocking progress here, I'd be ok with keeping this with a "TODO".  For example:

```python
plt.close('all') # TODO this is a bandaid, should instead make sure all other tests properly close their plots
```

If these plots are left over from the test_ProcessSofastFixed.py, then maybe add a tear down method.  For example:

```python
@classmethod
def tearDownClass(cls):
    plt.close('all')
```

Reply:
As usual, @bbean23, you gave a robust solution that works very well. I implemented the tearDownClass method.